vendor-local/src/legal-docs

```diff
@@ -1 +1 @@
-Subproject commit 0b9d56b979421eb9606949ca3dfbd4140950e107
+Subproject commit a41df77591b5f4aba6b736410ecc94a81add29ee
```

Comment:
This sub module commit seems to have been added by accident and should be removed.